tests/v2/test_rest_debug_api.nim

```diff
@@ -0,0 +1,51 @@
+import
+  stew/shims/net,
+  chronicles,
+  testutils/unittests,
+  presto,
+  libp2p/crypto/crypto
+import
+  ../../waku/v2/node/wakunode2,
+  ../../waku/v2/node/rest/[server, client, debug_api]
+
+
+proc testWakuNode(): WakuNode = 
+  let rng = crypto.newRng()
+  let privkey = crypto.PrivateKey.random(Secp256k1, rng[]).tryGet()
+  let bindIp = ValidIpAddress.init("0.0.0.0")
+  let extIp = ValidIpAddress.init("127.0.0.1")
+  let port = Port(9000)
```

Comment:
How do you feel about block `let` statements, e.g.:

```suggestion
  let
    rng = crypto.newRng()
    privkey = crypto.PrivateKey.random(Secp256k1, rng[]).tryGet()
    bindIp = ValidIpAddress.init("0.0.0.0")
    extIp = ValidIpAddress.init("127.0.0.1")
    port = Port(9000)
```

Reply:
I am ok with them. I guess that it's just that I am not used to this Nim particularity 🙂